js/packages/components/chat/message/shared-components/User.tsx

```diff
@@ -0,0 +1,132 @@
+import React from 'react'
```

Comment:
I think we can flaten the tree by moving what's in "shared-components" to the parent folder